hystrix/callback/callback.go

```diff
@@ -0,0 +1,34 @@
+package callback
+
+var circuitCallback map[string]stateFunc
+
+//State is a type to hold Circuit-state this will be used while calling stateFunc on State change
+type State string
+
+const (
+	//Open is a state to indicate that Circuit state is Open
+	Open = "Open"
+	//Close is a state to indicate that Circuit state is Close
+	Close = "Close"
+	//AllowSingle is a state to indicate that Circuit state is AllowSingle or trying to Open Circuit
```

Comment:
AllowSingle is for trying to 'close' the circuit?

Reply:
Yes